bridge-proto/proto/query.proto

```diff
@@ -38,6 +38,11 @@ enum Consistency {
   LOCAL_ONE = 0x0A;
 }
 
+enum ResumeMode {
+  NEXT_ROW = 0x00;
```

Comment:
Is the paging state data different for `NEXT_ROW` vs. `NEXT_PARTITION` or is it the same state data?